### PR TITLE
Replace requests aiohttp

### PR DESCRIPTION
**Pull request by Azrood**
Finally done with this ! It got harder in header.py but I did it in the end ^_^

### CODE REVIEW

**Comment by Azrood** [comment]
Resolved some merge conflicts

File: utils/urban.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """Module to search on urban dictionary."""
+import aiohttp
```
Comment (Sergeileduc) [inline utils/urban.py]: import but not used

File: utils/bonjourmadame.py [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
-def latest_madame():
+async def latest_madame():
     """Fetch last Bonjourmadame picture."""
     madames = "http://feeds2.feedburner.com/BonjourMadame"
-
-    res = requests.get(madames)
-    res.close()
-    soup = BeautifulSoup(res.text, 'lxml')
-
+    async with aiohttp.ClientSession() as session:
```
Comment (Sergeileduc) [inline utils/bonjourmadame.py]: Why not use 'get_soup_lxml()' from tools ?

**Review by Sergeileduc — CHANGES_REQUESTED** [review]
Awesome !
Seems good !

I just made two comments (import unused and some code that you should replace with function)

I requested this changes, before merging.

See yah

File: utils/urban.py (hunk 1 above)
Comment (Azrood) [reply utils/urban.py]: Yep, forgot about it.

File: utils/bonjourmadame.py (hunk 2 above)
Comment (Azrood) [reply utils/bonjourmadame.py]: Yeah seems good, I didn't really think about it, I just changed `requests` to `aiohttp` 😅

**Comment by Azrood** [comment]
Implemented the requested changes. See yah !

**Review by Sergeileduc — APPROVED** [review]

**Comment by Sergeileduc** [comment]
Ok !
Squash merge ! (squash make only one commit in the history.
Not always good, but I show you that)
Congrats !

**Pull request merged**